phpunit.xml

```diff
@@ -14,7 +14,8 @@
     <testsuites>
         <testsuite name="Application Test Suite">
             <directory suffix="Test.php">./tests/Legacy</directory>
-            <directory suffix="Test.php">./testsorchestra</directory>
+            <directory suffix="Test.php">./tests/Regression</directory>
+            <directory suffix="Test.php">./tests/Orchestra</directory>
```

Comment:
Will the system tests be going into a suite of their own?

Reply:
I'm not sure how we will hook them up would depend on what's easier for us and Travis

Reply:
- Check models are well-formed
- Check migrations are well-formed and idiomatic
- Verify PHPUnit hookup